README.md

```diff
@@ -1,8 +1,14 @@
 # ${GITHUB_REPOSITORY_NAME}
```

Comment:
This needs instructions for an analyst on what to do once they clone the template

E.g. When you first clone you should change {github-reposiroy-name} to x
You may want to link to other information about your project for example if you have standalone website
Once your study is published it is a good idea to update the repo with DOI for your paper so interested readrs can read more


 Suggest these are written in italics - and first bit of advice is. Once you have tweaked the information you should delete the instructions in italics